config/routes.rb

```diff
@@ -2,4 +2,14 @@
   devise_for :users
   root to: 'pages#home'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :spots, :only [:index, :show] do
+    resources :sessions, :only []
```

Comment:
blank because we don't have the controller?
should include new, create, index, edit, update